.github/workflows/reusable_deploy_docs.yml

```diff
@@ -92,14 +97,29 @@ jobs:
           process_gcloudignore: false
           parent: false
 
-      - name: "Upload Python Docs (stable)"
+      - name: "Update latest"
         if: ${{ inputs.UPDATE_LATEST }}
-        uses: google-github-actions/upload-cloud-storage@v2
-        with:
-          path: "rerun_py/site"
-          destination: "rerun-docs/docs/python/stable"
-          process_gcloudignore: false
-          parent: false
+        run: |
+          cat <<EOF > index.html
+          <!DOCTYPE html>
+          <html>
+          <head>
+            <meta charset="utf-8">
+            <title>Redirecting</title>
+            <noscript>
+              <meta http-equiv="refresh" content="1; url=../${{ inputs.PY_DOCS_VERSION_NAME }}/" />
+            </noscript>
+            <script>
+              window.location.replace("../${{ inputs.PY_DOCS_VERSION_NAME }}/" + window.location.hash);
+            </script>
+          </head>
+          <body>
+            Redirecting to <a href="../${{ inputs.PY_DOCS_VERSION_NAME }}/">../${{ inputs.PY_DOCS_VERSION_NAME }}/</a>
+          </body>
+          </html>
+          EOF
+
+          gsutil cp ./index.html gs://rerun-docs/docs/python/stable/
```

Comment:
this python doc stuff seems unrelated to js docs. Do I read this correctly that you're fixing the latest redirect here instead of reuploading the python docs yet-another-time?